utils/cards.ts

```diff
@@ -23,3 +28,44 @@ export const updateCardsStorage = () =>
     .then(() => exec(["cd data/cards", "git pull"].join("\n")))
     .then(() => exec("git log -n 1"))
     .then(data => data.stdout);
+
+export const isCardTypeExists = (() => {
+  const memory: {
+    [key: string]: boolean;
+  } = {};
+  return (cardType: string) => {
+    if (cardType in memory) {
+      return memory[cardType];
+    }
+    try {
+      require(`card-types/types/${cardType}`);
+      memory[cardType] = true;
+    } catch (e) {
+      memory[cardType] = false;
+    }
+    return memory[cardType];
+  };
+})();
```

Comment:
What is the reason of using here an immediately invoked function expression?

Reply:
memoization - I've created a local scope for `memory` variable